wger.py

```diff
@@ -42,62 +52,86 @@ def post_info(self,url_file,post_data):
     def delete_info(self,url_file):
         """General function for DELETE requests"""
         delete_url = self.base_url + url_file
-        requests.delete(url=delete_url,headers={'Authorization': self.auth_token})
-        return "Item Deleted"
+        response=requests.delete(url=delete_url,headers={'Authorization': self.auth_token})
+        return response
+
+    def match(self,dict1,dict_item,param1,param2):
+        current_value=""
+        for i in dict1:
+            if i[param1] == dict_item:
+                current_value = i.get(param2)
+        return current_value
 
-    def chose_weight(self,date,weight):
+
+    def weight_entry(self,date,weight):
         """The function creates weight goals for a specific date."""
 
-        data={ "id":"","date":date,"weight":weight,"user":""}
-        self.post_info('weightentry/',data)
-        return "Weight goal added successfully"
+        data={ "id":"",
+               "date":date,
+               "weight":weight,
+               "user":""}
+        response=self.post_info('weightentry/',data)
```

Comment:
After you implement the response code check for each request, check them here as well and treat the cases(if the call is done correctly and if not).